rdm/0004-browser-support.md

```diff
@@ -0,0 +1,98 @@
+- Start Date: 2019-10-02
+- RFC PR: [5](https://github.com/inveniosoftware/rfcs/pull/5)
+- Authors: Guillaume Viger
+
+# Support Evergreen Browsers
+
+## Summary
+
+InvenioRDM commits to supporting secure, evergreen browsers that are themselves
+supported by their vendors.
+
+## Terminology
+
+Evergreen Browser: A browser that keeps itself up-to-date without prompting the user.
+
+## Motivation
+
+- As an InvenioRDM core or extension/plugin developer, I want to take advantage
```

Comment:
Hmm We are using `InvenioRDM` and `Invenio RDM` interchangably. I think it was agreed to use `Invenio RDM`. @lnielsen Could you confirm?

Reply:
Was wondering about that too. Will wait for confirmation.